restli-server/src/test/java/com/linkedin/restli/server/test/TestRestLiDefaultInResponse.java

```diff
@@ -0,0 +1,126 @@
+/*
+   Copyright (c) 2020 LinkedIn Corp.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package com.linkedin.restli.server.test;
```

Comment:
Move to `com.linkedin.restli.internal.server.response` to be aligned with the location of `ResponseUtils`